http/controllers/task.controller.ts

```diff
@@ -2,17 +2,31 @@
 
 import { Request, Response } from "express";
 import { TasksServiceInstance } from "../../src/application";
+import { appDevelopmentLogger } from "../../src/common";
 import { JSON_MESSAGES } from "./utils";
 class TaskController {
   async fetchUserTasksList(req: Request, res: Response) {
     const { userId } = req.params;
-    const tasks = await TasksServiceInstance.getList(100, 0, userId);
+    if (!userId) {
+      return res
+        .status(400)
+        .json({ success: false, message: JSON_MESSAGES.BAD_REQUEST });
+    }
+    const limit = parseInt(req.body.limit) || 10;
+    const page = parseInt(req.body.page) || 1;
+    const tasks = await TasksServiceInstance.getList(limit, page, userId);
     return res.status(200).json({ success: true, tasks });
   }
 
   async addUserTask(req: Request, res: Response) {
     const { userId } = req.params;
+    if (!userId) {
+      return res
+        .status(400)
+        .json({ success: false, message: JSON_MESSAGES.BAD_REQUEST });
+    }
     const task = await TasksServiceInstance.create(req.body, userId);
+    appDevelopmentLogger({ task }, { context: "Task Create Controller" });
```

Comment:
Controllers should be lean, use logger in the services instead.

Reply:
resolved